### DISCUSSION TITLE
SOC console URL Base

### DISCUSSION THREAD

**Discussion by bit1290** [Security-Onion-Solutions/securityonion#7312] [Unsupported Versions] [answered]
Hello,

I have an IP to access SOC web console:

![image](https://user-images.githubusercontent.com/16745902/155521497-7e55e585-7a32-4d70-ad9e-a9d60750dd3c.png)

configured on url_base in global.sls.

Is it possible to have more than one?

Thank you :)

**Comment by dougburks** [comment, marked as answer]
You can only have one `url_base` but you can set it to a hostname so that it can resolve to different IP addresses if necessary:
https://docs.securityonion.net/en/2.3/url-base.html
